recipe_batches/recipe_batches.py

```diff
@@ -3,12 +3,16 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+    batches = []
+    for ing in recipe.keys():
+        ingredients.setdefault(ing, 0)
```

Comment:
very nice! I didn't know of setdefault()
all() also works great here
https://www.programiz.com/python-programming/methods/built-in/all